.github/workflows/pre-commit-linter.yml

```diff
@@ -44,3 +44,20 @@ jobs:
 
       - name: Run pre-commit
         uses: pre-commit/action@v3.0.0
+
+  prettier-linter:
+    name: prettier-linter
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '22'
```

Comment:
Any specific reason we are using 22 instead of 20 here? I saw we use 20 in other places: https://github.com/PelicanPlatform/pelican/blob/1ec45a9618e4a64991230a5504650ed013e01fc1/.github/workflows/test.yml#L21